eks-aperf.sh

```diff
@@ -174,7 +174,7 @@ fi
 
 # Show resource usage for pods on this node
 echo -e "${BOLD}Resource usage for pods on ${NODE_NAME}:${NC}"
-rm  /tmp/allpods.out 2> /dev/null;  \
+rm  /tmp/allpods.out 2> /dev/null || true
```

Comment:
We don't need this line at all. The next line overwrites the file unconditionally.